experiments/targeting.go

```diff
@@ -189,7 +189,8 @@ func NewEqualNode(inputNodes map[string]interface{}) (Targeting, error) {
 func (n *EqualNode) Evaluate(inputs map[string]interface{}) bool {
 	candidateValue := inputs[n.fieldName]
 	switch n := candidateValue.(type) {
-	case int, float64:
+	case int, float64, int8, int16, int32, int64, uint, uint8, uint16, uint32,
+		uint64, float32:
```

Comment:
If you're going to do linebreaks, do it to group things together logically, not based on length:

```suggestion
	case
		float32, float64,
		int, int8, int16, int32, int64,
		uint, uint8, uint16, uint32, uint64:
```